000-debug/logger.php

```diff
@@ -137,11 +140,11 @@ function lo( $stuff, ...$rest ) {
 function l_json_encode_pretty( $data ) {
 	if ( defined( 'JSON_PRETTY_PRINT' ) ) {
```

Comment:
Since we don't run PHP older than 7.4, I suggest getting rid of the legacy code and remove the `else` branch in another PR, if there are no objections.